drivers/sensor/bq274xx/bq274xx.h

```diff
@@ -15,27 +15,27 @@
 #define BQ274XX_DEVICE_ID 0x0421 /* Default device ID */
 
 /*** Standard Commands ***/
-#define BQ274XX_COMMAND_CONTROL_LOW 0x00 /* Control() low register */
-#define BQ274XX_COMMAND_CONTROL_HIGH 0x01 /* Control() high register */
-#define BQ274XX_COMMAND_TEMP 0x02 /* Temperature() */
-#define BQ274XX_COMMAND_VOLTAGE 0x04 /* Voltage() */
-#define BQ274XX_COMMAND_FLAGS 0x06 /* Flags() */
-#define BQ274XX_COMMAND_NOM_CAPACITY 0x08 /* NominalAvailableCapacity() */
-#define BQ274XX_COMMAND_AVAIL_CAPACITY 0x0A /* FullAvailableCapacity() */
-#define BQ274XX_COMMAND_REM_CAPACITY 0x0C /* RemainingCapacity() */
-#define BQ274XX_COMMAND_FULL_CAPACITY 0x0E /* FullChargeCapacity() */
-#define BQ274XX_COMMAND_AVG_CURRENT 0x10 /* AverageCurrent() */
-#define BQ274XX_COMMAND_STDBY_CURRENT 0x12 /* StandbyCurrent() */
-#define BQ274XX_COMMAND_MAX_CURRENT 0x14 /* MaxLoadCurrent() */
-#define BQ274XX_COMMAND_AVG_POWER 0x18 /* AveragePower() */
-#define BQ274XX_COMMAND_SOC 0x1C /* StateOfCharge() */
-#define BQ274XX_COMMAND_INT_TEMP 0x1E /* InternalTemperature() */
-#define BQ274XX_COMMAND_SOH 0x20 /* StateOfHealth() */
-#define BQ274XX_COMMAND_REM_CAP_UNFL 0x28 /* RemainingCapacityUnfiltered() */
-#define BQ274XX_COMMAND_REM_CAP_FIL 0x2A /* RemainingCapacityFiltered() */
-#define BQ274XX_COMMAND_FULL_CAP_UNFL 0x2C /* FullChargeCapacityUnfiltered() */
-#define BQ274XX_COMMAND_FULL_CAP_FIL 0x2E /* FullChargeCapacityFiltered() */
-#define BQ274XX_COMMAND_SOC_UNFL 0x30 /* StateOfChargeUnfiltered() */
+#define BQ274XX_COMMAND_CONTROL_LOW 0x00	/* Control() low register */
```

Comment:
To reduce line lengths I would suggest to rename these macros with these substring replacements:
COMMAND -> CMD
EXTENDED -> EXT
CONTROL -> CTRL

Possibly you could also remove the BQ274XX prefix from these macros if they were moved into ba274xx.c .

Reply:
ok, good point!